.github/workflows/dapr-test.yml

```diff
@@ -106,6 +106,9 @@ jobs:
       - name: Deploy dapr to ${{ env.TEST_CLUSTER }} cluster
         if: env.TEST_CLUSTER != ''
         run: make docker-deploy-k8s
+      - name: Deploy default config
+        if: env.TEST_CLUSTER != ''
```

Comment:
Why do we need to turn off mtls during test?

Reply:
We don't have to. I think we might want to do it in stages through, first enabling it for e2e test for mTLS, and once we see that's stable we enable it for all the tests.